ugvc/mrd/mrd_utils.py

```diff
@@ -169,10 +169,18 @@ def collect_coverage_per_locus(coverage_bw_files, df_sig):
     return df_list
 
 
+def collect_coverage_per_locus_gatk(coverage_csv, df_sig):
```

Comment:
Please add a docstring

Reply:
Also, I suggest more general names like "vcf_dataframe" or "signature_dataframe" for clarity